health/micro-ui/web/packages/modules/campaign-manager/src/pages/employee/NewAppConfiguration/NewLayoutRenderer.js

```
@@ -0,0 +1,246 @@
+          onFieldClick && onFieldClick(field, data, null, index, null);
+        }}
+        style={{
+          cursor: "pointer",
+          border: isSelected ? "2px solid #0B4B66" : "2px solid transparent",
+          borderRadius: "4px",
+          padding: "8px",
+          margin: "4px 0",
+          backgroundColor: isSelected ? "#f0f8ff" : "transparent",
+          transition: "all 0.2s ease",
+        }}
+      >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)